bittensor/_neuron/text/core_server/__init__.py

```diff
@@ -79,6 +79,17 @@ def __init__(
         if config == None: config = server.config()
         config = config; 
 
+        config.netuid = netuid if netuid != None else config.netuid
+
+        subtensor = bittensor.subtensor ( config = config ) if subtensor == None else subtensor
```

Comment:
This is changed for #1053, lets merge your PR first @camfairchild, I will add this to that pr

Reply:
@Eugene-hu merged